### ISSUE TITLE
Trackpad not working on Dell Latitude 7480

### ISSUE THREAD

**Issue by stossy11** [blankmac/AlpsHID#23]
just installed both kext and they are in my config.plist but my trackpad isn't working keyboard is working I'm on this rn

**Comment by Lorys89** [comment]
Se dsdt amd ioreg log

**Comment by stossy11** [comment]
its working but only sometimes what is dsdt amd ioreg log?

**Comment by KojoAnti** [comment]
> its working but only sometimes what is dsdt amd ioreg log?

Please I'm using same laptop, my Trackpard isn't also working

**Comment by KojoAnti** [comment]
Did you a find a solution?

**Comment by stossy11** [comment]
Yes I had just copied the kexts and the SSDTs updated all the kexts and it worked + added the patches from the config plist

**Comment by Lalacol1993** [comment]
hello I am having the same issue but I don't understand your last post. are there ssts that we need specifically for this to work? also by patches in the config.plist do you mean oc snapshot?
